benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/BroadcastBenchmark.cs

```diff
@@ -49,11 +49,36 @@ public void GlobalSetup()
                 var hubConnection = new HubConnectionContext(connection, Timeout.InfiniteTimeSpan, NullLoggerFactory.Instance);
                 hubConnection.Protocol = protocol;
                 _hubLifetimeManager.OnConnectedAsync(hubConnection).GetAwaiter().GetResult();
-
-                _ = ConsumeAsync(connection.Application);
+                _connections.Add(connection);
+                _hubConnections.Add(hubConnection);
+                //_ = ConsumeAsync(connection.Application);
             }
 
             _hubContext = new HubContext<Hub>(_hubLifetimeManager);
+
+            _hubContext.Clients.All.SendAsync("Method").GetAwaiter().GetResult();
+        }
+
+        [GlobalCleanup]
+        public void GlobalCleanup()
```

Comment:
I don't think it's correct to do this after, I think you do want to kick off consumption during creation because that mimicks how it works in reality.